libs/long.js

```diff
@@ -309,6 +342,15 @@
     *     if the given one is greater.
     */
     Long.prototype.compare = function (other) {
```

Comment:
Can you add some simple tests for the fast paths in *compare*, *add* and *shiftRightUnsigned* to gnu.testlet.vm.LongTest?